pandas/core/indexes/category.py

```diff
@@ -253,9 +253,8 @@ def get_values(self):
         """ return the underlying data as an ndarray """
         return self._data.get_values()
 
-    def __iter__(self):
-        """ iterate like Categorical """
-        return self._data.__iter__()
+    def tolist(self):
+        return self._data.tolist()
```

Comment:
I don't think this is needed? Base `Index` already defines this as `self._values.tolist()`, and `_values` and `_data` is both a Categorical ?